src/kit/Form.tsx

```diff
@@ -27,7 +27,7 @@ interface FormItemProps {
   extra?: ReactNode;
   field?: AntdFormListFieldData;
   hidden?: boolean;
-  initialValue?: string | number | Primitive;
```

Comment:
@johnkim-det do you remember the reason why the type is `string | number | Primitive` instead of `Primitive`? is it ok to be just `Primitive | Primitive[]`?

Reply:
I don't think that was intentional, should be fine to change